### PR TITLE
Add Blinq icon

### LINKED ISSUE #10763: Request: Blinq
### Brand Name

Blinq

### Website

https://blinq.me/

### Popularity Metric

Ranked 72,813 globally on Similarweb. See https://www.similarweb.com/website/blinq.me/#overview.

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by uncenter**
blinq

**Issue:** closes #10763.

**Popularity metric:** Ranked 72,813 globally on Similarweb. See https://www.similarweb.com/website/blinq.me/.

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I linked to the Drive folder here which I'm not sure if goes against our guidelines on sources (https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines). The press kit is linked in the footer of the website but it links straight to the Drive folder and there is no intermediate page.

### CODE REVIEW

**Review by service-paradis — CHANGES_REQUESTED** [review]
I left a few comments, but I'm not even sure we can this one according to https://blinq.me/legal/website-terms-of-use

> Subject to the above, your use of, and access to, the Site and the Content does not grant or transfer to you any rights, title or interest to Our Intellectual Property. Unless otherwise permitted in these Terms, you must not:
> copy or use, in whole or in part, any of Our Intellectual Property;

File: icons/blinq.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Blinq</title><path d="M12 20.2667c2.2091 0 4-1.7908 4-4h-.8c0 1.7673-1.4327 3.2-3.2 3.2s-3.2-1.4327-3.2-3.2v2.4004c.7298.9713 1.8915 1.5996 3.2 1.5996m0-8c2.2091 0 4 1.7908 4 4h-.8c0-1.7674-1.4327-3.2-3.2-3.2s-3.2 1.4326-3.2 3.2v-2.4004c.7298-.9714 1.8915-1.5996 3.2-1.5996m0-.8c2.2091 0 4-1.7909 4-4h-.8c0 1.7673-1.4327 3.2-3.2 3.2s-3.2-1.4327-3.2-3.2V9.867c.7298.9713 1.8915 1.5996 3.2 1.5996m0-8c2.2091 0 4 1.7909 4 4h-.8c0-1.7674-1.4327-3.2-3.2-3.2s-3.2 1.4326-3.2 3.2V5.0662c.7298-.9713 1.8915-1.5996 3.2-1.5996M8 0h.8v24H8Z"/></svg>
```
[the remark this answers is not in the document]
Comment (uncenter) [reply icons/blinq.svg]: Sounds good. I've updated the icon.

**Comment by uncenter** [comment]
New preview:
image

**Comment by adamrusted** [comment]
I'd say the guidelines posted in https://github.com/simple-icons/simple-icons/pull/10816#pullrequestreview-2003892041 give us a pretty clear-cut answer. Sorry @uncenter!

**Comment by uncenter** [comment]
> I'd say the guidelines posted in [#10816 (review)](https://github.com/simple-icons/simple-icons/pull/10816#pullrequestreview-2003892041) give us a pretty clear-cut answer. Sorry @uncenter!

No worries!

**Pull request closed without merging**